charts/databend-query/templates/configmap.yaml

```diff
@@ -89,3 +89,11 @@ data:
         {{- end }}
         {{- end }}
       {{- end }}
+
+    [cache]
```

Comment:
maybe we could make this group of value deeper like `.Values.config.storage.s3`

Reply:
you're right, just added, plz review.